src/modules/ordersTable/pure/OrdersTableContainer/OrdersTable.tsx

```diff
@@ -308,6 +309,20 @@ export function OrdersTable({
     checkbox.checked = allOrdersSelected
   }, [allOrdersSelected, selectedOrders.length])
 
+  const [collapsedOrder, setCollapsedOrder] = useState<{ [index: string]: boolean }>({})
+
+  const handleOnClickToggle = (order: ParsedOrder) => {
+    const orderId = order.composableCowInfo?.id // If it's a parent, use its own ID
```

Comment:
Not a comment for you, for for @shoom3301, but I wonder in the future if we have again this pattern of parent/child relationship (we will for other smart orders), we have here a bunch of logic that depends on `composableCowInfo`, so we will likely need to refactor this parent/child logics a bit